tools/unicodeCat.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const request = require('request');
+
+create(
+    'bmpLetters',
+    'https://raw.githubusercontent.com/mathiasbynens/unicode-6.2.0/' +
```

Comment:
Right, the build process should not need external resources at every run. Come to think of it, I should probably design some way of offline mode, where it won't run things like `npm install`, so that the build can be used when not hooked up to the internet at all even if the `package.json` changes due to git branch hopping. But that's another story.